docs/CONNECTOR_PROTOCOL.md

```diff
@@ -277,6 +277,10 @@ In addition to the connector index `.elastic-connectors`, we have an additional
       service_type: string;   -> Service type of the connector
     }
   ];
+  checkpoint: {
```

Comment:
In Workplace Search, this is called `cursors`.

Reply:
True, but I'm not fully sure if `cursors` is better than `checkpoint`. What do you think about it?

Reply:
I don't have a preference. Just a reminder in case you forgot.

Please use whichever you feel it's more appropriate.